src/context/DataContext.jsx

```diff
@@ -171,12 +172,19 @@ export const DataProvider = ({ children }) => {
                 'broadcast',
                 { event: 'frame-changed' },
                 (payload) => {
-                    console.log(payload)
                     data.renderingEngine.getViewport(payload.payload.viewport).setImageIdIndex(payload.payload.frame)
                     data.renderingEngine.getViewport(payload.payload.viewport).render()
                 }
             )
 
+            interaction_channel.on(
```

Comment:
Will switch this to instead be handled via a useEffect that relies on the shareController presence object to see if the session owner's stack object is different and use that to trigger rather than via a broadcast message.